governance/proposals/012-set-protocol-fee-cross-bridge.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * This proposal send calls accross the bridge to enable protocol fees
+ * on all supported networks.
+ */
+const { ADDRESS_ZERO, getNetwork } = require('@unlock-protocol/hardhat-helpers')
+const { Unlock } = require('@unlock-protocol/contracts')
+const { IConnext, targetChains } = require('../helpers/bridge')
+const { parseSafeMulticall } = require('../helpers/multisig')
+const { ethers } = require('hardhat')
+const PROTOCOL_FEE_STRING = '0.0001'
+
+const parseSetProtocolFeeCalls = async (destChainId) => {
+  const { unlockAddress } = await getNetwork(destChainId)
+
+  const protocolFee = ethers.parseEther(PROTOCOL_FEE_STRING)
+
+  console.log(`Proposol to set protocolFee to ${ethers.formatEther(protocolFee)}
+    - unlock : ${unlockAddress}`)
+
+  // get Unlock interface
+  const { interface: unlockInterface } = await ethers.getContractAt(
+    Unlock.abi,
+    unlockAddress
+  )
+
+  // parse unlock calls
+  const calls = [
+    {
+      contractAddress: unlockAddress,
+      calldata: unlockInterface.encodeFunctionData('setProtocolFee', [
+        protocolFee,
+      ]),
+      value: 0,
+      operation: 1, // Unlock is a proxy so use DELEGATECALL
+    },
+  ]
+
+  const oldProtocolFee = await unlockInterface.protocolFee()
+  const explainers = [
+    `- Protocol fee for ${unlockAddress} set to ${PROTOCOL_FEE_STRING}
+  \`setProtocolFee(${protocolFee})\` (Previous Protocol Fee: ${ethers.formatEther(
+    oldProtocolFee
+  )})`,
+  ]
+
+  return {
+    calls,
+    explainers,
+  }
+}
+
+const parseSafeCall = async ({ destChainId, calls }) => {
+  // parse multicall
+  const { to, data, value, operation } = await parseSafeMulticall({
```

Comment:
Here there is a single call for each chain so this step can be skipped. (This was used to pack multiple call into a single one)